src/app/not-found.tsx

```diff
@@ -0,0 +1,87 @@
+"use client";
+import { Button, Container, Typography, Box } from '@mui/material';
+import { useRouter } from 'next/navigation';
+import Image from 'next/image';
+import { AUTH_STATUS, AUTH_TOKEN } from '../shared/constants/storage';
+import { useEffect, useState } from 'react';
+
+export default function NotFoundPage() {
+    const router = useRouter();
+
+    const [loggedIn, setLoggedIn] = useState(false);
+
+    useEffect(() => {
+        const token = typeof window !== 'undefined' ? localStorage.getItem(AUTH_TOKEN) : null;
```

Comment:
1. I have refactor the frontend's some variables, please use the new variables after I Merging to dev. AUTH_TOKEN to ACCESS_TOKEN.
2. I changed the logic of auth status. In previous, you trigger a router, if the refresh token is invalid, the response interceptor in the Apollo client will save the status into localStorage, and then after you triggering another a router, it will push to login page. But now, it will redirect directly in the Apollo client, so the AUTH_STATUS is redundant now.